base/server/src/main/java/org/dogtagpki/server/rest/ACLInterceptor.java

```diff
@@ -194,14 +195,16 @@ else if (principal instanceof GenericPrincipal) {
 
         // If still not available, it's unprotected, allow request.
         if (!authzRequired) {
-            logger.debug("ACLInterceptor: No ACL mapping; authz not required.");
+            logger.debug("ACLInterceptor: Unprotected resource; access granted");
 
             auditor.log(AuthzEvent.createSuccessEvent(
                         auditSubjectID,
                         null, //resource
                         null, //operation
                         LOGGING_MISSING_ACL_MAPPING + ":" + auditInfo)); //info
```

Comment:
I understand this is existing code, but I wonder why we decided to audit record authz events that have no actual authorization being done.  If we simply want to allow admins to record such events, then we should at least create a different audit event so that such records could be differentiated (filtered) from the ones that did require authz to minimize the amount of records being written.

Reply:
I guess it depends on the semantics. In this case the user is "authorized" to access an unprotected resource (e.g. retrieving the list of certs in CA). If we don't record this, there won't be any audit log for the execution (or the authorization) of the operation. But if you think this should not be considered an authorization we can remove this too, just let me know.